pg-index-health-testing/src/main/java/io/github/mfvanek/pg/testing/PostgreSqlClusterAliasHolder.java

```diff
@@ -77,11 +84,17 @@ WaitStrategy getWaitStrategyForStandBy() {
     }
 
     @Nonnull
-    private Map<String, String> createCommonEnvVarsMap() {
+    private Map<String, String> createCommonEnvVarsMap(
+            @Nonnull final String customUsername,
+            @Nonnull final String customPassword
+    ) {
+        final String username = Objects.requireNonNull(customUsername, "username mustn't be null");
```

Comment:
@Evreke 
I use `cannot` instead of `mustn't` in the project. I fix it in your PR.
Also I always cover such cases with unit test.